assets/accessories/games.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Keep the directory name meaningful. accessories is not the right place for this. Create a directory called topbar or menu or icons